### PR TITLE
Add columns Post CSS function for on grid width calculations

### PR DESCRIPTION
**Pull request by joe-hastings**
Add a Post CSS function that gives on grid width calculations for neater CSS so we don't need to have the long calc all over our code.

### CODE REVIEW

**Comment by joe-hastings** [comment]
I suppose I could also go through and replace all the places widths that should be on grid are used to use this?

File: {{cookiecutter.repo_name}}/gulpfile.js/lib/postCssProcessors.js [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
   require('postcss-nested'),
   require('postcss-functions')({
-    glob: path.join(__dirname, '../../{{cookiecutter.package_name}}', 'assets', 'css', 'helpers', 'functions', '*.js')
+    functions: {
+      percentage: function (val1, val2) {
+        let number = val1 / val2 * 100
+
+        return number.toFixed(2) + '%'
+      },
+      columns: function (n, t, gutter) {
```
Comment (lewiscollard) [inline {{cookiecutter.repo_name}}/gulpfile.js/lib/postCssProcessors.js]: Can you change `n` and `t` to full English words?

**Review by lewiscollard — CHANGES_REQUESTED** [review]

**Review by joe-hastings — COMMENTED** [review]
It's a bit odd how we use 8 columns at `--sm` and 12 at `--md`, we don't really tend to do this in new projects its always 12 all the way down to mobile, might be worth updating this?

**Comment by lewiscollard** [comment]
Feel free to change that @joe-hastings, if you want. All but one of these are `width: columns(6, 8);`, which can be expressed as `columns(9, 12)` anyway.

**Comment by joe-hastings** [comment]
> Feel free to change that @joe-hastings, if you want. All but one of these are `width: columns(6, 8);`, which can be expressed as `columns(9, 12)` anyway.

Cool think I will cause it's just easier to see whats going on at a quick glance, I can't be doing complex arithmetic like that on the spot ;)

**Review by lewiscollard — APPROVED** [review]

**Pull request merged**